src/tests/features/rich_tcp_mss_clamp.at

```diff
@@ -0,0 +1,98 @@
+dnl ###############
+dnl ## XML tests ##
+dnl ###############
+
+FWD_START_TEST([tcp-mss-clamp])
+AT_KEYWORDS(tcp-mss-clamp)
+
+AT_CHECK([mkdir -p ./zones])
+
+AT_DATA([./zones/tcp.xml], [dnl
+<?xml version="1.0" encoding="utf-8"?>
+<zone>
+  <service name="ssh"/>
+  <service name="dhcpv6-client"/>
+  <rule>
+    <tcp-mss-clamp value="536"/>
+  </rule>
+</zone>
+])
+FWD_CHECK([--check-config],0, ignore)
+
+AT_DATA([./zones/tcp.xml], [dnl
+<?xml version="1.0" encoding="utf-8"?>
+<zone>
+  <service name="ssh"/>
+  <service name="dhcpv6-client"/>
+  <rule>
+    <tcp-mss-clamp value="pmtu"/>
+  </rule>
+</zone>
+])
+FWD_CHECK([--check-config],0, ignore)
+
+AT_DATA([./zones/tcp.xml], [dnl
+<?xml version="1.0" encoding="utf-8"?>
+<zone>
+  <service name="ssh"/>
+  <service name="dhcpv6-client"/>
+  <rule>
+    <tcp-mss-clamp value="thisdoesnotexist"/>
+  </rule>
+</zone>
+])
+m4_ifdef([TESTING_FIREWALL_OFFLINE_CMD], [FWD_CHECK([--check-config], 0, ignore, [dnl
+WARNING: INVALID_RULE: thisdoesnotexist: rule tcp-mss-clamp value="thisdoesnotexist"
+WARNING: INVALID_RULE: thisdoesnotexist: rule tcp-mss-clamp value="thisdoesnotexist"
+])], [FWD_GREP_LOG([dnl
+WARNING: INVALID_RULE: thisdoesnotexist: rule tcp-mss-clamp value="thisdoesnotexist"
+])])
+
+AT_DATA([./zones/tcp.xml], [dnl
+<?xml version="1.0" encoding="utf-8"?>
+<zone>
+  <service name="ssh"/>
+  <service name="dhcpv6-client"/>
+  <rule>
+    <tcp-mss-clamp/>
+  </rule>
+</zone>
+])
+FWD_CHECK([--check-config], 0, ignore,ignore)
+
+dnl start cli tests (permanent configuration)
+AT_DATA(rm ./zones/tcp.xml)
+FWD_CHECK([--permanent --add-rich-rule='rule tcp-mss-clamp value=thisdoesnotexist'],122,ignore,ignore)
+FWD_CHECK([--permanent --add-rich-rule='rule tcp-mss-clamp'],0,ignore, ignore)
+FWD_CHECK([--permanent --add-rich-rule='rule tcp-mss-clamp value=0'],122,ignore, ignore)
+FWD_CHECK([--permanent --add-rich-rule='rule tcp-mss-clamp value=536'],0,ignore, ignore)
+FWD_CHECK([--permanent --add-rich-rule='rule tcp-mss-clamp value=pmtu'],0,ignore, ignore)
+
+dnl start runtime tests for tcp/mss clamp
+FWD_CHECK([--add-rich-rule='rule tcp-mss-clamp value=pmtu'],0,ignore, ignore)
+FWD_CHECK([--add-rich-rule='rule tcp-mss-clamp value=0'],122,ignore, ignore)
+FWD_CHECK([--add-rich-rule='rule tcp-mss-clamp'],0,ignore, ignore)
```

Comment:
Can you verify doing a `--delete-rich-rule` or `--query-rich-rule` still works if `value` is omitted? It's worth adding coverage for these.